canonical-web-and-design/ubuntu.com
# use article card template on engage index

## Done

- moved re-used markup on /engage into a partial, making it easier to add new entries.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/engage
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page looks identical to https://ubuntu.com/engage
